helm/cluster-cloud-director/values.yaml

```diff
@@ -97,4 +97,7 @@ userContext:
 # cluster-shared configuration
 includeClusterResourceSet: "true"  # must be a string
 # let's use the kubectl image defaults
-kubectlImage: {}
+kubectlImage:
+  registry: quay.io
+  name: giantswarm/kubectl
+  tag: 1.23.5
```

Comment:
I think we should add a comment that it's used in `cluster-shared`. Otherwise it's very cryptic.